src/blob/persistence/LokiBlobMetadataStore.ts

```diff
@@ -95,27 +107,51 @@ import { getBlobTagsCount, getTagsFromString, toBlobTags } from "../utils/utils"
  * @class LokiBlobMetadataStore
  */
 export default class LokiBlobMetadataStore
-  implements IBlobMetadataStore, IGCExtentProvider {
+  implements IBlobMetadataStore, IGCExtentProvider
+{
   private readonly db: Loki;
 
   private initialized: boolean = false;
   private closed: boolean = true;
 
+  private readonly accountModelFromArgs: AccountModel | undefined;
+
+  private accountModel: AccountModel | undefined;
```

Comment:
Please check will blob meta data size saved on disk increase a lot?

Reply:
I've seen the expected increase, which is that new version create more entries, but the entries just have the few versionId related fields (mainly ID and current version). The other overhead is the account model, but people making 100s of accounts would be very unlikely.